### DISCUSSION TITLE
上传OneDrive能不走服务器吗？国内的服务器上传太难受了

### DISCUSSION THREAD

**Discussion by gongjing666** [AlistGo/alist#1054] [Ideas]
希望大佬能开发一下。

**Comment by fzlychee2019** [comment]
这和ALIST没啥关系吧？把安装ALIST的服务器翻墙不就解决问题了？

**Reply by gongjing666** [reply]
我用的是世纪互联

**Comment by fzlychee2019** [comment]
上传问题，官方有在常见问题里解答过，看来短时间内，恐怕官方是不会去解决这个问题的。末将期待能直接上传到云盘，减轻服务器负担。

**Reply by gongjing666** [reply]
昨晚上我也去看了一下文档，说实在的，网盘走服务器中转是真的牛逼，国内服务器那点带宽还不如不开发上传功能

**Reply by xhofe** [reply]
> 昨晚上我也去看了一下文档，说实在的，网盘走服务器中转是真的牛逼，国内服务器那点带宽还不如不开发上传功能

你可以选择不用。

**Reply by anwen-anyi** [reply]
> 昨晚上我也去看了一下文档，说实在的，网盘走服务器中转是真的牛逼，国内服务器那点带宽还不如不开发上传功能

🥶上传大文件你直接用官方客户端上传不香么，知道Alist上传比较慢还用那你说怎么办…😂还有就是你用的世纪互联直接本地上传多快吖…

**Reply by sakurayun** [reply]
> 昨晚上我也去看了一下文档，说实在的，网盘走服务器中转是真的牛逼，国内服务器那点带宽还不如不开发上传功能

哪家网盘挂载od，上传不走服务器啊😓
